app/scripts/services/metricsCharts.js

```diff
@@ -80,11 +82,6 @@ angular.module("openshiftConsole")
                 left: 0,
                 top: 20,
                 bottom: 0
-              },
-              tick: {
-                format: function(value) {
-                  return d3.round(value, 3);
-                }
```

Comment:
Since we're letting c3.js pick the tick values, this is no longer necessary. c3.js picks round numbers. It can also cause problems for very low CPU usage where a tick is repeated.